### PR TITLE
Fixed Meta, Keyword out of Sync Issues

### PR DESCRIPTION
**Pull request by VigneshMurugan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,9 @@
     "type": "git",
     "url": "git://github.com/GrosSacASac/DOM99.git"
   },
-  "dependencies": {},
+  "dependencies": {
+    "handlebars": "^4.0.12"
```
Comment (GrosSacASac) [inline package.json]: make it a dev dependency

**Comment by VigneshMurugan** [comment]
Done Please check

File: package.json (hunk 1 above)
Comment (VigneshMurugan) [reply package.json]: Done Please check

**Comment by GrosSacASac** [comment]
Nice

**Pull request merged**